src/App.css

```diff
@@ -39,134 +32,297 @@ a {
 
 .Nav-parent-container {
   display: flex;
-  margin: 2% auto 0;
-  width: 50%;
+  justify-content: flex-start;
+  background-color: white;
+  padding: 1rem 2rem;
+  border-bottom: 1px solid #ccc;
+  gap: 2rem;
+  padding-left: 7.5%;
   align-items: center;
-  justify-content: space-between;
+  width: 100%;
+  margin: 0 auto;
 }
```

Comment:
- [ ] I think it would be better to use `#e8e8e8` color for `border-bottom` as given in the Ziplin guidelines 😃